README.md

```diff
@@ -1,92 +1,102 @@
+* [How to process one pixel](examples/basic/python_script/pixel_ticoi_demo.py)
 
-**- python_script**
-* **examples/basic/python_script/ticoi_cube_demo.py**: Demonstration of how to process one cube
-* **examples/basic/python_script/ticoi_pixel_demo.py**: Demonstration of how to process one pixel
+### Advanced examples
 
+* [How to process one ITS_LIVE cube directly from the cloud](/examples/advanced/cube_ticoi_demo_its_live.py)
+* [How to format several geotiff files into a netCDF file](examples/advanced/cube_prep_from_geotiff.py)
+* [How to apply GLAFT on TICOI results](examples/advanced/glaft_for_ticoi_results.py)
 
-#### More advanced examples
+## TO USE YOUR ONE DATASET
 
-* **cube_ticoi_demo_its_live.py**: Demonstration of how to process one ITS_LIVE cube directly from the cloud
-* **examples/advances_processing/cube_ticoi_demo_advanced.py**: Demonstration of how to process one cube, by using different parameters for different areas (stable, surge)
-* **examples/data_availability.py**: A few tools to evaluate the data availability on a set of data, its repartition throughout the period of measurement and its quality
-in some cases, in order to see whether it is relevant or not to study glacier velocities at a point (seaonality, surge...). It uses
-TICOI's optimized loading to load the data.
-* **examples/optimize_coef/pixel_optimize_coef_nvcc.py**: Demonstration of how to choose the regularisation parameter according to the Vector Velocity Coherence
+### You have geotiff files
 
-### TO START
+You need to convert them into netcdf, by
+modified [this script](examples/advanced/cube_prep_from_geotiff.py).
 
-#### You have geotiff files
-You need to convert them into netcdf, for that have a look at advanced_processing/geotiff_to_xarray.py
+### You have netcdf files
 
-#### You have netcdf files
 If it is ITS_LIVE data, or Millan et al., 2019, you can directly use them!
-If not, you have to create your own dataloader, within which the dimension should be ("mid_date", "y", "x"), and the variables should be "vx", "vy", and should contain the projection information in the ds.proj4 attribute.
+If not, you have to create your own dataloader, within which the dimension should be ("mid_date", "y", "x"), and the
+variables should be "vx", "vy", and should contain the projection information in the ds.proj4 attribute.
 
-### OUTPUTS
-* to understand to output of pixel_demo please check README_visualization_pixel_output.md
+## HYPERPARAMETERS AND OUTPUTS
 
+* to understand to output of pixel_demo please
+  check [README_output](README_output.md)
+* to understand the parameters you can change, please
+  check [README_possible_parameters](README_possible_parameters.md)
 
-<p align="center">
-  <img src="examples/image/TICOI_workflow.png" alt="TICOI Workflow" width="800"/>
-</p>
+
+## TO CONTRIBUTE
+
+If you have an idea to improve this package, you are welcome to contribute! See guidelines [here](CONTRIBUTING.md). 
```

Comment:
an idea to -> an idea how to
or simply: "We welcome any contribution to this package! See guidelines..."